docs/faq.md

```diff
@@ -0,0 +1,68 @@
+# F.A.Q.
+
+Here are a couple of common questions about JupyterLab and the transition from
+the classic Notebook interface.
+
+## Is the classic notebook going away ?
+
+No, at least not yet, it will though get way less updates. There will be a
+transition period from Classic notebook to JupyterLab. Currently the notebook
+server you get when you install Jupyter comes with Classic Notebook
+preinstalled. JupyterLab is an extension. When you install JupyterLab, the
+notebook server runs both classic notebook and JupyterLab. 
+
+When JupyterLab has been widely adopted, the server will come with Lab by
+default, and the classic notebook will be an extension. 
+
+## What will happen to my notebook files ?
+
+JupyterLab still use the same file format. You can open the same notebook both
+in JupyterLab and classic notebook. Notebook will still work on nbviewer, be
+rendered on GitHub... etc. JupyterLab is just a new interface. 
+
+## I like the notebook interface, I do not want to migrate to JupyterLab.
+
+The notebook interface is still present in JupyterLab, and unlike in classic
+notebook you can drag and drop between notebooks, have multiple notebook side
+by side, and a couple of other features not possible in classic notebook.
+JupyterLab also support more file formats. For example you can execute
+code in a markdown document or render GeoJson. Some libraries may need some time
```

Comment:
~~Some libraries...issue.~~

New paragraph:

For most users, the transition to JupyterLab should be seamless. As always, please feel free to ask questions and open issues if needed.